db/migrate/20220901175043_create_regular_transactions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRegularTransactions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :regular_transactions, id: :uuid do |t|
+      t.belongs_to :gross_income_summary, foreign_key: true, null: false, type: :uuid
```

Comment:
Personally I feel the consistency with other existing tables is significant  nd these are all using the  model/ORM dependant: destroy mechanism, but I also, more generally, rather like the flexibility of enforcement via the model/ORM rather than the database (which i have seen cause issues in the vague distant past)